src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -136,24 +132,22 @@ function LHNOptionsList({
         ({item: reportID}) => {
             const itemFullReport = reports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
             const itemReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
-            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`];
-            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`];
-            const itemTransaction = `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(
-                itemParentReportActions,
-                [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'],
-                '',
-            )}`;
+            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`] || {};
+            const itemParentReportAction = itemParentReportActions[itemFullReport.parentReportActionID] || {};
+            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`] || {};
+            const transactionID = lodashGet(itemParentReportAction, [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'], '');
```

Comment:
```suggestion
            const transactionID = lodashGet(itemParentReportAction, ['originalMessage', 'IOUTransactionID'], '');
```

Had to watch this with 🦅's eye 😄

Reply:
oh wow, the 🦅 eye indeed 🤯